site/src/pages/WorkspacesPage/WorkspacesPage.tsx

```diff
@@ -38,19 +23,17 @@ const WorkspacesPage: FC = () => {
       </Helmet>
 
       <WorkspacesPageView
-        filter={workspacesState.context.filter}
-        isLoading={!workspaceRefs}
-        workspaceRefs={workspaceRefs}
-        count={count}
-        getWorkspacesError={getWorkspacesError}
-        onFilter={(query) => {
-          send({
-            type: "UPDATE_FILTER",
-            query,
-          })
+        workspaces={data?.workspaces}
+        error={error}
+        filter={filter.query}
+        onFilter={filter.setFilter}
+        count={data?.count}
+        page={pagination.page}
+        limit={pagination.limit}
+        onPageChange={pagination.goToPage}
+        onUpdateWorkspace={(workspace) => {
+          updateWorkspace.mutate(workspace)
```

Comment:
Curious about the separation of `WorkspacesPage` and `WorkspacesPageView`. I notice some of the layout is in the parent component, and some is in the child  - should we have the parent handle all the page layout, e.g. margins, page headers, subtitles?
I also see we're passing a lot of props down from the parent still - is this to facilitate testing or do we like that pattern for another reason?

Reply:
I have no strong opinion on this, I kept what we were using, to be honest. But, in the current state, I would like to only keep data logic in the Page component and all the visuals in PageView. I think in this way, we can test more logic in the stories. What about you?

Reply:
If it keeps our stories running, I'm all for it.